README.md

```diff
@@ -133,6 +133,30 @@ wget -q https://raw.githubusercontent.com/AUTOMATIC1111/stable-diffusion-webui/m
 ```
 3. Run `webui.sh`.
 4. Check `webui-user.sh` for options.
+
+### Automatic Installation on Linux (Ubuntu container w/ GPU)
+
+By default the `webui.sh` script requires GPU access during installation, so building a Ubuntu container takes a few additional steps:

```

Comment:
I think you'll just want to add `--skip-torch-cuda-test`.

Reply:
Cool yeah that allows the build to be done without GPU, so I changed the `Dockerfile` to perform the install and pass the `--exit` flag, obviating the needs for the extra steps I had originally.

However, now the SD model is downloaded every-time the container starts. It seems the `HF_HOME` env var is not respected, and I assume the `--exit` is preventing that download from occurring (perhaps I'm wrong and there's an easy workaround). 

Any ideas how we can cache that download even when using the `--exit` flag to setup the environment?